tests/encodings.rs

```diff
@@ -0,0 +1,87 @@
+use quick_xml::events::Event;
+use quick_xml::Reader;
+
+#[cfg(feature = "encoding")]
```

Comment:
You should instead of this flag mark these tests as requiring `encoding` feature in `Cargo.toml`, like that is done for `serde-de.rs` tests